src/components/Input/Input.vue

```diff
@@ -156,6 +156,10 @@ export default defineComponent({
         ? ['hovered']
         : []
     },
+    attributes() {
+      const { ...attrs } = this.$attrs
```

Comment:
We need to remove `class` and `style` from `$attrs` so they don't get bound to the input.  
```suggestion
      const { class, style, ...attrs } = this.$attrs
```